### ISSUE TITLE
AI Assistant view stays empty

### ISSUE THREAD

**Issue by JonasHelming** [eclipsesource/ide-ai-assistant#41]
Branch [main](https://github.com/eclipsesource/ide-ai-assistant)
Running:
yarn
yarn start
open app in a browser
open AI assistant view.

Output:
See attached photo, extension appears empty

![image](https://github.com/eclipsesource/ide-ai-assistant/assets/1052615/0846d82b-7e9f-498e-bf9b-825688729e81)

**Comment by JonasHelming** [comment]
@eneufeld Could you have a look at this please? This only occurs for one team member, but for him on Windows and Linux. I was also able to reproduce the issue on may machine.

**Comment by eneufeld** [comment]
this works for me.
Could you check and paste errors you have in the browser dev tools as well as on the console

**Comment by eneufeld** [comment]
and could you paste your npm and yarn versions

**Comment by eneufeld** [comment]
for me:
node: 18.16.1
yarn: 1.22.19

**Comment by JonasHelming** [comment]
@mathis-girault It works on firefox for me, can you check this? 
Might have something to do with this: An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.

**Comment by eneufeld** [comment]
Ok, this needs further investigation but:
using localhost:3000 everything should work
using 127.0.0.1:3000 is breaking

**Comment by eneufeld** [comment]
here is a detailed explanation: https://github.com/eclipse-theia/theia/discussions/10605#discussioncomment-1941074

Tl;dr; use localhost

**Comment by JonasHelming** [comment]
@eneufeld Afterthought: Do you know which component prints out the adress on the terminal? Maybe we can ensure localhost there too to avoid this error for other users
